doc/newsboat.asciidoc

```diff
@@ -1171,6 +1171,14 @@ Identifier:Meaning
 [[articleview-title-format-t]]<<articleview-title-format-t,+t+>>:Number of total articles
 |======================================================================
 
+.Search Result Title Format Identifiers
```

Comment:
I'd move this table up, right after the one for the article list, and added a note that all of the article list identifiers are supported here too.